src/main/java/com/openmrs/migrator/core/utilities/FileIOUtilities.java

```diff
@@ -156,4 +175,148 @@ public boolean removeDirectory(File directoryToBeDeleted) throws InvalidParamete
     }
     return directoryToBeDeleted.delete();
   }
+
+  public void addSettingToConfigFile(String dataBaseName) throws IOException {
```

Comment:
As already mentioned we will be using the settingService from ticket 100 for reading settings from a file etc... Please remove these and use that rather.

Best to keep this fileIO only functions here (read/write/etc...)

Reply:
done